server/sonar-server/src/main/java/org/sonar/server/issue/IssueQueryService.java

```diff
@@ -206,7 +209,7 @@ public IssueQuery createFromRequest(SearchWsRequest request) {
         request.getFileUuids(),
         request.getAuthors());
 
-      builder.createdAfter(buildCreatedAfterFromRequest(session, request, allComponentUuids, effectiveOnComponentOnly));
+      builder.createdAfter(buildCreatedAfterFromRequest(session, request, allComponentUuids));
```

Comment:
Great removal ! But could you quickly explain me how did you you manage to remove this 'effectiveOnComponentOnly' parameter ?